explorer/notebooks/cenus_exploration.ipynb

```diff
@@ -0,0 +1,2324 @@
+{
```

Comment:
- Please add titles and full legend to plots so that they are easier to use outside of the notebook

- Maybe install `seaborn` on the explorer docker directly?

_ Please try and keep the same ordering for each plots pop, white, asian, ...